openreview/openreview.py

```diff
@@ -200,7 +200,15 @@ def get_profile(self, email_or_id):
         profile = response.json()['profile']
         return Profile.from_json(profile)
 
-    def get_profiles(self, email_or_id_list, limit=1000):
+    def get_profiles_by_name(self, first, last):
```

Comment:
Could it be possible to use the same function signature? I mean to have a single 'get_profiles' call?

Reply:
I wrote this a while ago, and I think there was a reason I didn't do it that way, but I'm not certain. Let me try to make it work again, maybe it the problem will reveal itself (or maybe I'll be able to do it)